esmvalcore/cmor/check.py

```diff
@@ -539,6 +540,26 @@ def _check_coord_values(self, coord_info, coord, var_name):
                 self._cube.remove_coord(coord)
                 self._cube.add_aux_coord(new_coord, dims)
 
+    def _check_longitude_max(self, coord, var_name):
+        if not self.automatic_fixes:
+            return False
```

Comment:
yes, as @bouweandela says - needs a critical report here :beer:

Reply:
Fixed before the call